doc/howto/network_acls.md

```diff
@@ -639,21 +639,283 @@ Note that the request _inserted_ the new `user.limit` key without affecting the
+
+Set the network's `security.acls` to a string that contains the ACL name or names you want to add. Comma-separate multiple names:
+
+```bash
+lxc network set <network_name> security.acls="<ACL_name>[,<ACL_NAME>,...]"
+```
+
+#### Examples
+
+Set the `br01` network's `security.acls` to contain only the `web-traffic` ACL:
+
+```bash
+lxc network set br01 security.acls="web-traffic"
+```
+
+Set the `br01` network's `security.acls` to contain three ACLs:
+
+```bash
+lxc network set br01 security.acls="web-traffic,internal-traffic,ssh-access"
+```
+
+````
+% End of group-tab CLI
+
+````{group-tab} API
+
+Send a request to set the network's `security.acls` to a string that contains the ACL name or names you want to add. Comma-separate multiple names:
+
+```bash
+lxc query --request PATCH /1.0/networks/{networkName} --data '{
+  "config": {
+    "security.acls": "<ACL_name>[,<ACL_name>,...]"
+  }
+}'
+```
+
+See [the API reference](swagger:/networks/network_patch) for more information.
+
+#### Examples
+
+Set the `br01` network's `security.acls` to contain only the `web-traffic` ACL:
 
 ```bash
-lxc network set <network_name> security.acls="<ACL_name>"
+lxc network set br01 security.acls="web-traffic"
+```
+```bash
+lxc query --request PATCH /1.0/networks/br0 --data '{
+  "config": {
+    "security.acls": "web-traffic"
+  }
+}'
 ```
 
-For instance NICs, use the following command:
+Set the `br01` network's `security.acls` to contain three ACLs:
 
 ```bash
-lxc config device set <instance_name> <device_name> security.acls="<ACL_name>"
+lxc network set br01 security.acls="web-traffic,internal-traffic,ssh-access"
+```
+```bash
+lxc query --request PATCH /1.0/networks/br0 --data '{
+  "config": {
+    "security.acls": "web-traffic,internal-traffic,ssh-access"
+  }
+}'
 ```
 
+````
+% End of group-tab API
+
+`````
+% End of tabs
+
+### Assign an ACL to the NIC of an instance
+
+For {abbr}`NICs (Network Interface Cards)`, you can only assign an ACL to a NIC with {ref}`nic-ovn`.
+
+`````{tabs}
+````{group-tab} CLI
+
+To assign an ACL to an instance NIC, run:
+
+```bash
+lxc config device set <instance_name> <NIC_name> security.acls="<ACL_name>[,ACL_name,...]"
+```
+
+#### Example
+
+Assign a single ACL to an instance NIC:
+
+```bash
+lxc config device set ubuntu-container ovn-nic security.acls="web-traffic"
+```
+
+Assign multiple ACLs to an instance NIC:
+
+```bash
+lxc config device set ubuntu-container ovn-nic security.acls="web-traffic,internal-traffic,ssh-access"
+```
+
+````
+% End of group-tab CLI
+
+````{group-tab} API
+
+#### View the existing NIC configuration
+
+To update the configuration for an instance's device using the API, you must include all the required keys for the device—even if you're only changing one key. The required keys are the `type` of `nic` and the `network` name. Include these along with the key to update, and any other existing keys (unless you want to remove them). Omitted keys are reset to default values.
```

Comment:
This is starting to feel repetitive again. Especially as here we are crossing the boundary into the Instance Device realm. Can we document which config key needs updating for instance devices and then link to an instance device howto for updating devices which would have this warning?

Reply:
We dont want to start documenting instance device required keys outside of the instance or instance device pages, otherwise we are repeating ourselves.